source/install/upgrade_from_mongodb.rst

```diff
@@ -1,99 +1,100 @@
-      ii  mongodb-org-shell              3.6.2                              amd64        MongoDB shell client
-      ii  mongodb-org-tools              3.6.2                              amd64        MongoDB tools
-
-3. Remove installed packages:
+3. Remove the installed packages:
 
    .. code-block:: bash
 
-      apt-get remove mongodb-org mongodb-org-mongos mongodb-org-server \
-      mongodb-org-shell mongodb-org-tools
+      $ apt-get remove mongodb-org mongodb-org-mongos mongodb-org-server \
+      $ mongodb-org-shell mongodb-org-tools
 
-4. Install Percona Server for MongoDB :ref:`using apt <apt>`.
+4. Install |PSMDB| :ref:`using apt <apt>`.
 
 Upgrading on Red Hat Enterprise Linux or CentOS
-===============================================
-
-1. Stop the mongod process:
-
-   .. code-block:: bash
+================================================================================
 
-      $ service mongod stop
+|tip.run-all.root|
 
-2. Check for installed packages:
+1. Stop the mongod process: |service.mongod.stop|
+#. Check for installed packages: :bash:`rpm -qa | grep mongo`
 
-   .. code-block:: bash
+   .. admonition:: Output
 
-      $ rpm -qa | grep mongo
+      .. code-block:: guess
 
-      mongodb-org-mongos-3.6.2-1.el6.x86_64
-      mongodb-org-shell-3.6.2-1.el6.x86_64
-      mongodb-org-server-3.6.2-1.el6.x86_64
-      mongodb-org-tools-3.6.2-1.el6.x86_64
-      mongodb-org-3.6.2-1.el6.x86_64
+	 mongodb-org-mongos-4.0.4-1.el6.x86_64
+	 mongodb-org-shell-4.0.4-1.el6.x86_64
+	 mongodb-org-server-4.0.4-1.el6.x86_64
+	 mongodb-org-tools-4.0.4-1.el6.x86_64
+	 mongodb-org-4.0.4-1.el6.x86_64
 
-3. Remove installed packages:
+3. Remove the installed packages:
 
    .. code-block:: bash
 
-      yum remove \
-      mongodb-org-mongos-3.6.2-1.el6.x86_64 mongodb-org-shell-3.6.2-1.el6.x86_64 \
-      mongodb-org-server-3.6.2-1.el6.x86_64 mongodb-org-tools-3.6.2-1.el6.x86_64 \
-      mongodb-org-3.6.2-1.el6.x86_64
+      $ yum remove \
+      mongodb-org-mongos-4.0.4-1.el6.x86_64 \
+      mongodb-org-shell-4.0.4-1.el6.x86_64 \
+      mongodb-org-server-4.0.4-1.el6.x86_64 \
+      mongodb-org-tools-4.0.4-1.el6.x86_64 \
+      mongodb-org-4.0.4-1.el6.x86_64
 
 4. Install Percona Server for MongoDB :ref:`using yum <yum>`.
 
-.. note:: When you remove old packages,
-   your existing configuration file is saved
-   as :file:`/etc/mongod.conf.rpmsave`.
-   If you want to use this configuration with the new version,
-   replace the default :file:`/etc/mongod.conf` file.
-   For example, existing data may not be compatible
-   with the default WiredTiger storage engine.
+.. note::
+
+   When you remove old packages, your existing configuration file is saved as
+   :file:`/etc/mongod.conf.rpmsave`.  If you want to use this configuration with
+   the new version, replace the default :file:`/etc/mongod.conf` file.  For
+   example, existing data may not be compatible with the default WiredTiger
+   storage engine.
 
```

Comment:
please mention that following needs to be added into mongo.conf when upgrading from upstream:
```
processManagement:
  fork: true
  pidFilePath: /var/run/mongod.pid
```
Otherwise service won't start (for more info see: PSMDB-270)